src/env-utils.js

```diff
@@ -45,6 +45,23 @@ const envPath = async function( env ) {
     return envPath;
 };
 
+const parseEnvFromENV = async function() {
+    const envSlug = process.env.ENV;
+
+    if ( envSlug === undefined ) {
+        return false;
+    }
+
+    // Make sure that a .config.json file exists here
+    const configFile = path.join( await envPath( envSlug ), '.config.json' );
+
+    if ( ! await fs.exists( configFile ) ) {
```

Comment:
@ivankristianto looks like there is no async version of `exists` function in the `fs-extra` package. See here: https://github.com/jprichardson/node-fs-extra#async

I think it would be better to use `pathExists` method instead: https://github.com/jprichardson/node-fs-extra/blob/master/docs/pathExists.md

Reply:
@eugene-manuilov it exist in Node fs lib: https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_exists_path_callback
and I just realize the async func is deprecated now. Let me update the code based on your recommendation